mempool/policy.go

```diff
@@ -99,6 +99,11 @@ func checkInputsStandard(tx *btcutil.Tx, utxoView *blockchain.UtxoViewpoint) err
 		// they have already been checked prior to calling this
 		// function.
 		entry := utxoView.LookupEntry(txIn.PreviousOutPoint)
+		if entry == nil {
```

Comment:
Before this diff (as the comment notes), the inputs were already checked for existience here: 
https://github.com/btcsuite/btcd/blob/f7e9fba08600f151cf5d3f22cdcc5d50ed7c93b9/mempool/mempool.go#L1083-L1095

IIUC we're adding this as the check call may skip that check, in which case we should update the comment above (rationale for not checking nil there).

Reply:
oh I see. Yeah it has to do with the order of the checks. Now it's fixed in a previous commit to make sure this check is always performed before `checkInputsStandard`, thus this commit is no longer relevant.